src/utils/transform-filters.ts

```diff
@@ -15,8 +20,11 @@ function combineFiltersWithOR(filters: Filter[]): Filter | CombinedFilter {
 }
 
 /**
- * Convert a list of SelectableFilters use in Answers Headless
- * to a single nested filter stucture use in Answers Core
+ * Converts a list of {@link SelectableFilter}s used in Answers Headless to a
+ * single nested filter stucture used in Answers Core.
+ *
+ * @param selectableFilters - The filters to be transformed
+ * @returns The filters in a singly-nested structure
```

Comment:
can we link Filter and CombinedFilter here for the return structure?

Reply:
updated!